cli/internal/graph/graph.go

```diff
@@ -111,6 +120,14 @@ func (g *CompleteGraph) GetPackageTaskVisitor(
 		packageTask.LogFile = logFile
 		packageTask.Command = command
 
+		var passthrough env.EnvironmentVariablePairs
+		if taskDefinition.PassthroughEnv != nil {
+			passthroughEnvDetailedMap, err := env.GetHashableEnvVars(taskDefinition.PassthroughEnv, nil, "")
+			if err == nil {
+				passthrough = passthroughEnvDetailedMap.All.ToSecretHashable()
+			}
+		}
```

Comment:
We need to capture the `ToSecretHashable()` value of these to make them extra-useful in run summary instead of them just being a list of env var names.